aries_cloudagent/protocols/issue_credential/v2_0/routes.py

```diff
@@ -435,7 +436,7 @@ async def credential_exchange_list(request: web.BaseRequest):
     }
 
     try:
-        async with context.session() as session:
+        async with profile.session() as session:
             cred_ex_records = await V20CredExRecord.query(
                 session=session,
                 tag_filter=tag_filter,
```

Comment:
I can't post comments on lines you didn't change, but instances like this:            
`result = await _get_result_with_details(context.profile, cxr)`
could be replaced with just profile instead of context.profile

Reply:
Fixed in this MR [`89fc9e5` (#20)](https://github.com/globalid/aries-cloudagent-python/pull/20/commits/89fc9e5f5a0f4f3ae82c141932309b08f9c3aced)